packages/@atjson/util/src/index.ts

```diff
@@ -390,6 +400,24 @@ export function extractSlices(value: {
     });
   }
 
+  // Normalize the ranges again, as we may have extended ranges into each other
+  // First sort the ranges
+  rangesToDelete = rangesToDelete.sort(compareRanges).reduce((acc, range) => {
```

Comment:
I would prefer a for loop here for performance and clarity reasons, but I don't care that much